include/smack/SmackInstGenerator.h

```diff
@@ -30,6 +30,7 @@ class SmackInstGenerator : public llvm::InstVisitor<SmackInstGenerator> {
 
   Block* currBlock;
   llvm::BasicBlock::const_iterator nextInst;
+  llvm::Instruction* fstNonDebugInst;
```

Comment:
Please name this variable `firstNonDebugInst`.
Also, shouldn't you initialize it to null or something like that?

Reply:
Done. Yes, I should have initialized it to null.